recipes/i2c-tools/all/conanfile.py

```diff
@@ -0,0 +1,43 @@
+import os
+from conans import ConanFile, CMake, tools, AutoToolsBuildEnvironment
+
+
+class I2cConan(ConanFile):
+    name = "i2c-tools"
+    license = "LGPL 2.1"
+    author = "Eirik Brandtzæg eirikb@eirikb.no"
+    url = "https://mirrors.edge.kernel.org/pub/software/utils/i2c-tools/"
+    description = "I2C tools for the linux kernel as well as an I2C library."
+    topics = ("i2c")
+    os = "linux"
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"shared": [True, False], "fPIC": [True, False]}
+    default_options = {"shared": False, "fPIC": True}
+    generators = "cmake"
+
+    def validate(self):
+        if self.settings.os != "Linux":
+            raise ConanInvalidConfiguration("i2c-tools only support Linux")
+
+    @property
+    def _source_subfolder(self):
+        return "i2c-tools"
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        extracted_dir = "i2c-tools-{}".format(self.version)
+        os.rename(extracted_dir, self._source_subfolder)
+
+    def build(self):
+        self.run("make", cwd=self._source_subfolder)
```

Comment:
Please use the autotools build helper. It allows overriding the make program through `CONAN_MAKE_PROGRAM`.
```suggestion
        autotools = AutotoolsBuildEnvironment(self)
        autotools.make(target="libi2c.so" if self.options.shared else "libi2c.a")
        
```